python/ray/scripts/scripts.py

```diff
@@ -942,6 +948,7 @@ def timeline(address):
     type=str,
     help="Override the address to connect to.")
 def stat(address):
+    """Get the current metrics protobuf from a Ray cluster."""
```

Comment:
I don't understand this one. I understand the words, but is this approach documented anywhere?

Reply:
It's an internal debugging command. I updated the comment to reflect this.